op-e2e/system_test.go

```diff
@@ -658,6 +662,24 @@ func calcGasFees(gasUsed uint64, gasTipCap *big.Int, gasFeeCap *big.Int, baseFee
 	return x.Mul(x, new(big.Int).SetUint64(gasUsed))
 }
 
+// calcL1GasUsed returns the gas used to include the transaction data in
+// the calldata on L1
```

Comment:
What do you think about importing this from ref opti geth?

Reply:
I'll need to export the util from there. Thoughts @trianglesphere ?

Reply:
I'm fine leaving it as is. It's kinda nice to do, but means supporting the interface.
I suspect we'll eventually want to provide L1 Gas Interfaces publicly, but that can be dealt with later.